src/openrct2/ride/Vehicle.h

```diff
@@ -470,7 +475,8 @@ enum : uint32_t
     VEHICLE_SPRITE_FLAG_CORKSCREWS = (1 << 12),
     VEHICLE_SPRITE_FLAG_RESTRAINT_ANIMATION = (1 << 13),
     VEHICLE_SPRITE_FLAG_CURVED_LIFT_HILL = (1 << 14),
-    VEHICLE_SPRITE_FLAG_15 = (1 << 15),
+    // Used only on lifts (the transport ride), to only use 4 rotation sprites instead of 32.
+    VEHICLE_SPRITE_FLAG_ROTATION_COUNT_DIVIDE_BY_EIGHT = (1 << 15),
```

Comment:
Again i would call it 4_FLAT_ROTATION_FRAMES

Reply:
`VEHICLE_SPRITE_FLAG_4_ROTATION_FRAMES`